src/wifipaf/WiFiPAFConfig.h

```diff
@@ -0,0 +1,142 @@
+/*
+ *
+ *    Copyright (c) 2025 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file defines default compile-time configuration constants
+ *      for the CHIP PAFTP layer.
+ *
+ */
+
+#pragma once
+
+#include <system/SystemConfig.h>
+
+/**
+ *  @def WIFIPAF_LAYER_NUM_PAF_ENDPOINTS
+ *
+ *  @brief
+ *    This defines the number of WiFiPAFEndPoint objects allocated for use by the
+ *    WiFiPAFLayer subsystem. Value should be defined as the minimum of (max number
+ *    of simultaneous PAF connections the system supports, max number of
+ *    simultaneous PAF connections the application will establish).
+ */
+#ifndef WIFIPAF_LAYER_NUM_PAF_ENDPOINTS
+#define WIFIPAF_LAYER_NUM_PAF_ENDPOINTS 2
```

Comment:
OK, now I really don't get it.  If we support > 1 connection, how does any of the "shut down everything" logic I asked question about above make any sense?